controllers/numaresourcesscheduler_controller.go

```diff
@@ -60,6 +60,13 @@ type NUMAResourcesSchedulerReconciler struct {
 	ImagePullPolicy    corev1.PullPolicy
 }
 
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles,verbs=*
```

Comment:
we miss the configmaps permissions:
```
//+kubebuilder:rbac:groups="",resources=configmaps,verbs=*
```

Reply:
We have it in the kubeletconfig_controller and the permissions for all the controllers are rendered under the same ```role.yaml``` file

Reply:
sure. And each controller should add the annotations for each permission they need. Let `controller-gen` and the other automation coalesce them as needed.

Reply:
In second thought I'll add it in order to be more explicit